cvprac/cvp_api.py

```diff
@@ -3207,3 +3207,360 @@ def workspace_build_status(self, workspace_id, build_id):
             if self.clnt.apiversion >= 6.0:
                 return self.clnt.get(url, timeout=self.request_timeout)
         return self.clnt.get(url, timeout=self.request_timeout)
+
+
+    def get_change_control(self, cc_id, cc_time=None):
+        ''' Get the configuration and status of a change control using Resource APIs.
+
```

Comment:
so the name of the functions were chosen based on the[ resource API names](https://aristanetworks.github.io/cloudvision-apis/models/changecontrol.v1/) and also in a way to not coincide with the old and deprecated (or soon to be deprecated change control functions) . I mentioned in the docstring for all these if they are using resource API, but maybe we could make it somewhere else clear too, and maybe we could add into the docstring which CVP versions support what CC API, ideally people should read the documentation, I will also add a few labs and soon we will render the website too so hopefully it will be easier to understand what calls to use, what do you think?